### DISCUSSION TITLE
Impersonation attack on Telegram

### DISCUSSION THREAD

**Discussion by csagan5** [bromite/bromite#1186] [General]
In https://github.com/bromite/bromite/issues/1183 it was mentioned that someone tried to impersonate my identity and the Bromite project on Telegram.

See also: https://twitter.com/GrapheneOS/status/1398629441245532165

Note: I have never been on Telegram and all the official resources for the Bromite project are exclusively mentioned on the official website https://www.bromite.org/

**Comment by csagan5** [comment]
cc @anupritaisno1 @thestinger

**Comment by thestinger** [comment]
There was someone previously using your username (@csagan5). It might have been this same person. I suspect they might have realized their deception was going to fall apart and they switched to their own username but tried to continue keeping the feeling of clout and admin access they got from impersonating you. It's not entirely clear what was going on.

They're from the Techlore community and were spreading attacks from them on GrapheneOS. They kept pretending to be you in private, clearly attempting to spark a conflict between us. They temporarily left the Techlore channels in order to hide their participation there but they went back afterwards. It's part of a coordinated series of attacks from that community. It's pretty sad they're hurting other projects as collateral damage now too.

They had admin access in the unofficial Bromite room and others. I think most of that is fixed now. A lot of people now know this wasn't you. It's pretty weird stuff, honestly. In the future, people really need to verify these things better. It appears almost everyone assumed it was you based on others thinking it was and no one actually checked. I wasn't convinced it was you and I kept questioning it and the whole house of cards fell apart. They came pretty close to causing people supporting either project to end up in a conflict though. It was bad.

**Reply by csagan5** [reply]
> They're from the Techlore community and were spreading attacks from them on GrapheneOS. They kept pretending to be you in private, clearly attempting to spark a conflict between us.

If that is the motivation, I find it dumbfounding; but thanks for stepping in.

> It appears almost everyone assumed it was you based on others thinking it was and no one actually checked.

I am certain this is easy to pull off; I used to have lots of hope for Keybase to address/mitigate this problem.

> They came pretty close to causing people supporting either project to end up in a conflict though. It was bad.

This is sad.

I know I am not easy to reach (that is on purpose, I have limited time and online communities can become **toxic**) and perhaps someone is pissed off by how strict I am with issue submission, but that is why I opened up the GitHub discussions space (which is really not highly moderated, that is another activity I cannot commit to).

**Comment by sphinxcdi** [comment]
They also joined various channels to try push their false narrative of what happened. They leave out their impersonation of csagan5, of course:

* https://app.element.io/#/room/#CalyxOS:matrix.org/$cmIUplC-t4mjROVaFfKvtzXRl27N9nDNtC0-9FheeZE ([archive 1](https://archive.today/2021.05.30-201118/https://view.matrix.org/room/!aPqEsAdWuysydZNCic:matrix.org/?anchor=$v_MmTO8YEZwc-wI8dTmTOvBBB0DxFg08q9sZzs5gGLc&offset=660), [archive 2](https://archive.today/2021.05.30-224740/https://app.element.io/%23/room/%23CalyxOS:matrix.org/$cmIUplC-t4mjROVaFfKvtzXRl27N9nDNtC0-9FheeZE), [permalink](https://matrix.to/#/!aPqEsAdWuysydZNCic:matrix.org/$cmIUplC-t4mjROVaFfKvtzXRl27N9nDNtC0-9FheeZE?via=matrix.org&via=t2bot.io&via=privacytools.io)) — Search for `Nomino`, they changed their name at some point:  
> 𝙏𝙝𝙚 𝘼𝙣𝙩𝙞𝙢𝙖𝙩𝙩𝙚𝙧 changed their display name to Nomino.  
* https://t.me/techloregroup/455304 (their messages were deleted or removed, this link is only the context)

This is a consistent pattern from members/supporters of the Techlore community, it's sad that people believe and support this stuff.

----

https://t.me/joinchat/0y5l1eCfmaQ2Mzlk is another unofficial Bromite channel where this was discussed (see [context](https://t.me/c/1184755030/11270)).

**Comment by wchen342** [comment]
Are those people random community members, or somehow related to that Techlore project itself? If it is the latter I found it weird that the project lists both Bromite and GrapheneOS on their website and then tried to attack both, not sure I can understand what they are trying to achieve here.
I never used telegram so I didn't know this happened, but I guess I need to look out for similar situations in the future.

**Comment by thestinger** [comment]
Techlore as a project and as a community is highly malicious towards GrapheneOS and projects friendly towards it. They spent a long time pretending to be supportive while trying to undermine the project. The leader of the project, Henry, also known as Techlore himself, holds a massive grudge against GrapheneOS for not helping him make content for his YouTube channel and rejecting his demands to be made into our community manager with admin privileges in our Matrix rooms and on Reddit.

**Comment by thestinger** [comment]
I don't think they did this as an official action. Rather, they've spent a lot of effort encouraging their community to attack / harass us and their community now does that regularly. They're supportive of it and welcome many people in their community who have engaged in it. Henry himself has repeatedly made incredibly nasty and underhanded attacks on GrapheneOS. He spreads misinformation about it and directs both slander and bullying at our developers.

This incident is old news for us since they've made many attacks on our community / project / developers since this happened. Most of that hasn't involved Bromite, but the person who perpetrated this is still welcomed in the CalyxOS and Techlore communities as are the others.

**Comment by wchen342** [comment]
Ah I see, thanks for the information. I have recently discovered a similar pattern in another community and it is indeed alarming when something like this occurs.
Not much can be done here I suppose, and it is sad who has better access to medias has the say these days. Glad that you can control the damage to some extent.

**Comment by thestinger** [comment]
This appears to be continuing with other projects and people that are seen as tied to GrapheneOS in some way, unfortunately. I'll try to post more information soon when I find out more about what appears to be happening.

**Reply by csagan5** [reply]
Someone wrote [here](https://www.slant.co/options/37035/~bromite-review) that Bromite has spyware, it has no ad-blocking and it is unsecure. Two reviewers involved; probably unrelated to the attacks you mentioned, but consider yourself lucky that GrapheneOS/Vanadium are not listed there. Yet another potential source of _slander_.

**Reply by wchen342** [reply]
Maybe you already knew, but obviously the controversial has spread even to F-Droid, just discovered this: https://forum.f-droid.org/t/which-google-free-phone/8668/18

**Reply by thestinger** [reply]
By the way, the Narsil person in that thread is spreading misinformation about GrapheneOS and claiming our documentation says the opposite of what it does. Our documentation clearly states that we use HTTPS-based network time via time.grapheneos.org rather than the mix of carrier-based and NTP based like the stock OS, and also that we default to using our own connectivity check servers. I don't understand why people choose to spread this misinformation. They're doing the same spreading false claims about the baseband too, along with misrepresenting it as being any different than Wi-Fi, Bluetooth and other components. The same isolated component implements cellular, Wi-Fi and Bluetooth... and it's not as if Wi-Fi and Bluetooth work differently elsewhere.

**Reply by wchen342** [reply]
> I don't understand why people choose to spread this misinformation.

I will suggest you do not try to reason why they will do such things because you cannot beat them in their own games. Either stay away or if they do real damage, go with the formal procedural route. Maybe it is because I know things way scarier than what they did, but I would say that there are plenty of people that will happily do evil when paid enough/given some (imaginary) power/for zealotic reasons.

**Reply by spiderman973** [reply]
the peron who is saying that bromite is spyware and insecure He is the King of Misinformation
